app/controllers/api/resources_controller.rb

```diff
@@ -0,0 +1,135 @@
+module Api
+  class ResourcesController < ApplicationController
+    respond_to :json
+    
+    def index
+      @resources = {
```

Comment:
`/resources` should render all approved resources, we don't need to group them.  
 
JSON format should be like:
```
  {
   resources: [
      {id: , name: , resource_type:, cardinality: description:},
      ...
   ]
  }
```

`resource_type` will be translation, tafisr, script etc. And `cardinality_type` will be
`1_word`, `1_verse` etc